Xdebug: [Step Debug] Could not connect to debugging client. Tried: 127.0.0.1:9003 (through xdebug.client_host/xdebug.client_port) :-(
[31431:] Initialize tables (tbl_trials)
[31431:] Initialize tables (tbl_a,tbl_b,tbl_c,tbl_d) with 10 records

Xdebug: [Step Debug] Could not connect to debugging client. Tried: 127.0.0.1:9003 (through xdebug.client_host/xdebug.client_port) :-(
Xdebug: [Step Debug] Could not connect to debugging client. Tried: 127.0.0.1:9003 (through xdebug.client_host/xdebug.client_port) :-(
Xdebug: [Step Debug] Could not connect to debugging client. Tried: 127.0.0.1:9003 (through xdebug.client_host/xdebug.client_port) :-(
[31446:worker:2:b] Align for trial #1 circa 2022-09-01 00:10:56
[31449:worker:3:d] Align for trial #1 circa 2022-09-01 00:10:56
[31444:worker:1:a] Align for trial #1 circa 2022-09-01 00:10:56
[31449:worker:3:d] For trial #1, run task "worker:3:d" (sequence: tbl_d,tbl_c,tbl_b,tbl_a)
[31444:worker:1:a] For trial #1, run task "worker:1:a" (sequence: tbl_a,tbl_b,tbl_c,tbl_d)
[31446:worker:2:b] For trial #1, run task "worker:2:b" (sequence: tbl_b,tbl_a,tbl_c,tbl_d)
[31446:worker:2:b] FAILURE: In trial #1, task "worker:2:b" raised exception: PDOException: "SQLSTATE[40001]: Serialization failure: 1213 Deadlock found when trying to get lock; try restarting transaction"
#0 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/PdoDatabaseAdapter.php(51): PDO->exec("UPDATE tbl_a SET field_w2 = 'worker:2:b committed field_w2' WHERE id = 1")
#1 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/Hurtlocker.php(239): Hurtlocker\PdoDatabaseAdapter->execute("UPDATE tbl_a SET field_w2 = %1 WHERE id = %2", (Array:2))
#2 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/Hurtlocker.php(125): Hurtlocker\Hurtlocker->updateRecords((Array:4), (Array:1), "field_w2", "worker:2:b committed field_w2")
#3 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/PdoDatabaseAdapter.php(30): Hurtlocker\Hurtlocker->Hurtlocker\{closure}(1)
#4 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/Hurtlocker.php(123): Hurtlocker\PdoDatabaseAdapter->transact(Object(Closure), 1)
#5 phar:///Users/totten/bknix/bin/cv/src/Command/EvalCommand.php(52) : eval()'d code(1): Hurtlocker\Hurtlocker->worker(2)
#6 phar:///Users/totten/bknix/bin/cv/src/Command/EvalCommand.php(52): eval()
#7 phar:///Users/totten/bknix/bin/cv/vendor/symfony/console/Command/Command.php(255): Civi\Cv\Command\EvalCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 phar:///Users/totten/bknix/bin/cv/vendor/symfony/console/Application.php(1009): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 phar:///Users/totten/bknix/bin/cv/vendor/symfony/console/Application.php(273): Symfony\Component\Console\Application->doRunCommand(Object(Civi\Cv\Command\EvalCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 phar:///Users/totten/bknix/bin/cv/src/Application.php(59): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 phar:///Users/totten/bknix/bin/cv/vendor/symfony/console/Application.php(149): Civi\Cv\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 phar:///Users/totten/bknix/bin/cv/src/Application.php(26): Symfony\Component\Console\Application->run()
#13 phar:///Users/totten/bknix/bin/cv/bin/cv(27): Civi\Cv\Application::main("phar:///Users/totten/bknix/bin/cv/bin")
#14 /Users/totten/bknix/bin/cv(14): require("phar:///Users/totten/bknix/bin/cv/bin/cv")
#15 {main}

[31449:worker:3:d] FAILURE: In trial #1, task "worker:3:d" raised exception: PDOException: "SQLSTATE[40001]: Serialization failure: 1213 Deadlock found when trying to get lock; try restarting transaction"
#0 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/PdoDatabaseAdapter.php(51): PDO->exec("UPDATE tbl_c SET field_w3 = 'worker:3:d committed field_w3' WHERE id = 1")
#1 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/Hurtlocker.php(239): Hurtlocker\PdoDatabaseAdapter->execute("UPDATE tbl_c SET field_w3 = %1 WHERE id = %2", (Array:2))
#2 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/Hurtlocker.php(125): Hurtlocker\Hurtlocker->updateRecords((Array:4), (Array:1), "field_w3", "worker:3:d committed field_w3")
#3 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/PdoDatabaseAdapter.php(30): Hurtlocker\Hurtlocker->Hurtlocker\{closure}(1)
#4 /Users/totten/bknix/build/dmaster/web/s[31446:worker:2:b] Align for trial #2 circa 2022-09-01 00:11:00
ites/all/modules/civicrm/ext/hurtlocker/src/Hurtlocker.php(123): Hurtlocker\PdoDatabaseAdapter->transact(Object(Closure), 1)
#5 phar:///Users/totten/bknix/bin/cv/src/Command/EvalCommand.php(52) : eval()'d code(1): Hurtlocker\Hurtlocker->worker(3)
#6 phar:///Users/totten/bknix/bin/cv/src/Command/EvalCommand.php(52): eval()
#7 phar:///Users/totten/bknix/bin/cv/vendor/symfony/console/Command/Command.php(255): Civi\Cv\Command\EvalCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 phar:///Users/totten/bknix/bin/cv/vendor/symfony/console/Application.php(1009): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 phar:///Users/totten/bknix/bin/cv/vendor/symfony/console/Application.php(273): Symfony\Component\Console\Application->doRunCommand(Object(Civi\Cv\Command\EvalCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 phar:///Users/totten/bknix/bin/cv/src/Application.php(59): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 phar:///Users/totten/bknix/bin/cv/vendor/symfony/console/Application.php(149): Civi\Cv\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 phar:///Users/totten/bknix/bin/cv/src/Application.php(26): Symfony\Component\Console\Application->run()
#13 phar:///Users/totten/bknix/bin/cv/bin/cv(27): Civi\Cv\Application::main("phar:///Users/totten/bknix/bin/cv/bin")
#14 /Users/totten/bknix/bin/cv(14): require("phar:///Users/totten/bknix/bin/cv/bin/cv")
#15 {main}

[31449:worker:3:d] Align for trial #2 circa 2022-09-01 00:11:00
[31444:worker:1:a] Align for trial #2 circa 2022-09-01 00:11:00
[31449:worker:3:d] For trial #2, run task "worker:3:d" (sequence: tbl_d,tbl_c,tbl_b,tbl_a)
[31444:worker:1:a] For trial #2, run task "worker:1:a" (sequence: tbl_a,tbl_b,tbl_c,tbl_d)
[31446:worker:2:b] For trial #2, run task "worker:2:b" (sequence: tbl_b,tbl_a,tbl_c,tbl_d)
[31446:worker:2:b] FAILURE: In trial #2, task "worker:2:b" raised exception: PDOException: "SQLSTATE[40001]: Serialization failure: 1213 Deadlock found when trying to get lock; try restarting transaction"
#0 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/PdoDatabaseAdapter.php(51): PDO->exec("UPDATE tbl_a SET field_w2 = 'worker:2:b committed field_w2' WHERE id = 2")
#1 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/Hurtlocker.php(239): Hurtlocker\PdoDatabaseAdapter->execute("UPDATE tbl_a SET field_w2 = %1 WHERE id = %2", (Array:2))
#2 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/Hurtlocker.php(125): Hurtlocker\Hurtlocker->updateRecords((Array:4), (Array:1), "field_w2", "worker:2:b committed field_w2")
#3 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/PdoDatabaseAdapter.php(30): Hurtlocker\Hurtlocker->Hurtlocker\{closure}(2)
#4 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/Hurtlocker.php(123): Hurtlocker\PdoDatabaseAdapter->transact(Object(Closure), 2)
#5 phar:///Users/totten/bknix/bin/cv/src/Command/EvalCommand.php(52) : eval()'d code(1): Hurtlocker\Hurtlocker->worker(2)
#6 phar:///Users/totten/bknix/bin/cv/src/Command/EvalCommand.php(52): eval()
#7 phar:///Users/totten/bknix/bin/cv/vendor/symfony/console/Command/Command.php(255): Civi\Cv\Command\EvalCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 phar:///Users/totten/bknix/bin/cv/vendor/symfony/console/Application.php(1009): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 phar:///Users/totten/bknix/bin/cv/vendor/symfony/console/Application.php(273): Symfony\Component\Console\Application->doRunCommand(Object(Civi\Cv\Command\EvalCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 phar:///Users/totten/bknix/bin/cv/src/Application.php(59): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 phar:///Users/totten/bknix/bin/cv/vendor/symfony/console/Application.php(149): Civi\Cv\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 phar:///Users/totten/bknix/bin/cv/src/Application.php(26): Symfony\Component\Console\Application->run()
#13 phar:///Users/totten/bknix/bin/cv/bin/cv(27): Civi\Cv\Application::main("phar:///Users/totten/bknix/bin/cv/bin")
#14 /Users/totten/bknix/bin/cv(14): require("phar:///Users/totten/bknix/bin/cv/bin/cv")
#15 {main}

[31444:worker:1:a] FAILURE: In trial #2, task "worker:1:a" raised exception: PDOException: "SQLSTATE[40001]: Serialization failure: 1213 Deadlock found when trying to get lock; try restarting transaction"
#0 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/PdoDatabaseAdapter.php(51): PDO->exec("UPDATE tbl_c SET field_w1 = 'worker:1:a committed field_w1' WHERE id = 2")
#1 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/Hurtlocker.php(239): Hurtlocker\PdoDatabaseAdapter->execute("UPDATE tbl_c SET field_w1 = %1 WHERE id = %2", (Array:2))
#2 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/Hurtlocker.php(125): Hurtlocker\Hurtlocker->updateRecords((Array:4), (Array:1), "field_w1", "worker:1:a committed field_w1")
#3 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/PdoDatabaseAdapter.php(30): Hurtlocker\Hurtlocker->Hurtlocker\{closure}(2)
#4 /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/ext/hurtlocker/src/Hurtlocker.php(123): Hurtlocker\PdoDatabaseAdapter->transact(Object(Closure), 2)
#5 phar:///Users/totten/bknix/bin/cv/src/Command/EvalCommand.php(52) : eval()'d code(1): Hurtlocker\Hurtlocker->worker(1)
#6 phar:///Users/totten/bknix/bin/cv/src/Command/EvalCommand.php(52): eval()
#7 phar:///Users/totten/bknix/bin/cv/vendor/symfony/console/Command/Command.php(255): Civi\Cv\Command\EvalCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 phar:///Users/totten/bknix/bin/cv/vendor/symfony/console/Application.php(1009): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 phar:///Users/totten/bknix/bin/cv/vendor/symfony/console/Application.php(273): Symfony\Component\Console\Application->doRunCommand(Object(Civi\Cv\Command\EvalCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 phar:///Users/totten/bknix/bin/cv/src/Application.php(59): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 phar:///Users/totten/bknix/bin/cv/vendor/symfony/console/Application.php(149): Civi\Cv\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 phar:///Users/totten/bknix/bin/cv/src/Application.php(26): Symfony\Component\Console\Application->run()
#13 phar:///Users/totten/bknix/bin/cv/bin/cv(27): Civi\Cv\Application::main("phar:///Users/totten/bknix/bin/cv/bin")
#14 /Users/totten/bknix/bin/cv(14): require("phar:///Users/totten/bknix/bin/cv/bin/cv")
#15 {main}




